tests/system_tests/test_cli.py

```diff
@@ -28,25 +27,52 @@ def phospho_files():
     return pin, fasta
 
 
-@pytest.fixture
-def pepxml_file():
-    """Get the pepxml file"""
-    pepxml = Path("data", "msfragger.pepXML")
-    return pepxml
+def count_lines(path: Path):
+    """Count the number of lines in a file.
+
+    Parameters
+    ----------
+    path : Path
+        The path to the file.
+
+    Returns
+    -------
+    int
+        The number of lines in the file.
+    """
+    with open(path, "r") as file:
+        lines = file.readlines()
+    return len(lines)
 
 
 def test_basic_cli(tmp_path, scope_files):
     """Test that basic cli works."""
-    cmd = ["mokapot", scope_files[0], "--dest_dir", tmp_path]
-    subprocess.run(cmd, check=True)
-    assert Path(tmp_path, "mokapot.psms.txt").exists()
-    assert Path(tmp_path, "mokapot.peptides.txt").exists()
+    params = [scope_files[0], "--dest_dir", tmp_path]
+    run_mokapot_cli(params)
+    assert Path(tmp_path, "targets.psms").exists()
```

Comment:
@ezander pointed out that we changed the file suffix from before ".psms.txt" to just ".psms". The".txt" version more clearly informs the user what she can expect, which would be preferable. We would need to investigate which parts of the code base we need to revert.

Reply:
since you are doing that change, I would vote to make it even clearer. I think of .txt as space separated, we could go for a csv/tsv here (even .parquet would be viable IMO). @wfondrie LMK what you think!